include/picongpu/plugins/makroParticleCounter/PerSuperCell.hpp

```diff
@@ -106,21 +106,21 @@ namespace picongpu
         typedef MappingDesc::SuperCellSize SuperCellSize;
         typedef GridBuffer<size_t, simDim> GridBufferType;
 
-        MappingDesc* cellDescription;
         std::string notifyPeriod;
         std::string m_filenameExtension = openPMD::getDefaultExtension();
         std::string m_filenameInfix = "_%06T";
 
         std::string pluginName;
         std::string pluginPrefix;
         std::string foldername;
+        MappingDesc* cellDescription;
+
         mpi::MPIReduce reduce;
 
         std::unique_ptr<GridBufferType> localResult;
 
         // @todo upon switching to C++17, use std::option instead
         std::unique_ptr<::openPMD::Series> m_Series;
-        // set attributes for datacollector files
```

Comment:
Why did you remove this comment @psychocoderHPC?

Reply:
*offline discussion with @psychocoderHPC:*
This comment was a long-time-ago left-over. Should have been removed before.